blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -89,401 +33,93 @@ export class RunDetailsPipeline extends Component {
             document.addEventListener('keydown', this._handleKeys, false);
```

Comment:
Per @kzantow 's comment about scroll up not canceling karaoke, I think the problem is that attaching the `wheel` event to the scrollArea ref is that it will not get events if the user's cursor is to the right of the log console. Maybe attach this one to `document` too?